google/cloud/security/scanner/pipelines/base_data_pipeline.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Base data pipeline skeleton."""
+
+import abc
+
+from google.cloud.security.common.util import log_util
+
+LOGGER = log_util.get_logger(__name__)
+
+
+class BaseDataPipeline(object):
```

Comment:
Pending carise's comment, but perhaps we can have more appropriate naming here?

Instead of ```pipeline```, how about ```scanner```? 
And should use just drop ```data```?
How does this look?
```
google/cloud/security/scanner/scanners
google/cloud/security/scanner/scanners/base_scanner.py
google/cloud/security/scanner/scanners/iam_rules_scanner.py
```